src/js/tokens.js

```diff
@@ -0,0 +1,3 @@
+export default {
+    wbKey:"d6131e220048fb5a7677f283582c82db992866d5"
+}
```

Comment:
Adding it like this will expose the key to the world... we need to think of a better way.